packages/fxa-auth-server/lib/payments/paypal/helper.ts

```diff
@@ -35,10 +35,12 @@ type PaypalHelperOptions = {
 };
 
 type AgreementDetails = {
-  status: 'active' | 'cancelled';
   city: string;
   countryCode: string;
+  firstName: string;
+  lastName: string;
   state: string;
+  status: 'active' | 'cancelled';
```

Comment:
Are we using this anywhere?

Reply:
Dunno, I was just sorting the fields alphabetically.